### PR TITLE
Support older DOE PIV certificates

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR adds older CA certificates that are required to verify older Department of Energy PIV certificates.

## 💭 Motivation and context ##

We are working with some users who have older (early 2019) DOE PIVs and their certificates could not be verified until we added the appropriate CA certificates to our trust store.

## 🧪 Testing ##

This change was validated in Staging by running the updated `fetch_user_ca_certs.sh` to pull in the new CA certificates and hash them for use for OpenVPN.  Then (_and this part is quite important_), the VPN service was restarted so that the new CA certificates were picked up and used.  After that, our intrepid DOE PIV user was able to successfully connect to our VPN.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
Some great detective work!

File: files/fetch_user_ca_certs.sh [hunk 1]
```diff
@@ -32,7 +35,7 @@ fetch() {
       # Certificate is already in PEM format.
       cp "${source_file}" cert-a
     fi
-  elif [[ $url =~ .p7b$ ]]; then
+  elif [[ $url =~ .p7[bc]$ ]]; then
```
Comment (felddy) [inline files/fetch_user_ca_certs.sh:38]: 💪 🪄

**Review by felddy — APPROVED** [review]
> Then (and this part is quite important), the VPN service was restarted

🤣 
Great job!  🌟

**Review by mcdonnnj — APPROVED** [review]
💪💪💪

**Pull request merged**